static/app/views/settings/project/projectOwnership/ownershipRulesTable.spec.tsx

```diff
@@ -37,6 +37,27 @@ describe('OwnershipRulesTable', () => {
     expect(screen.getByText(user1.name)).toBeInTheDocument();
   });
 
+  it('should filter codeowners rules without actor names', () => {
+    const rules: ParsedOwnershipRule[] = [
+      {
+        matcher: {pattern: 'pattern', type: 'path'},
+        // @ts-expect-error this seems to only happen when adding a codeowners file
+        owners: [{type: 'user', id: user1.id, name: undefined}],
```

Comment:
Would it make sense to also have one here whose name is defined, and show that that one comes through but the one with no name doesn't?